observability/otel/otel-configuration-of-instrumentation.adoc

```diff
@@ -6,9 +6,9 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+The {OTELName} Operator uses a custom resource definition (CRD) file that defines the configuration of the instrumentation.
```

Comment:
@IshwarKanse, should this be like this?
 ```suggestion
The {OTELName} Operator uses an `Instrumentation` custom resource definition (CRD) file that defines the configuration of the instrumentation.
```

Reply:
And I have a question about this line (not a change in this PR, but is published as the intro paragraph in https://81817--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-configuration-of-instrumentation.html#otel-instrumentation-options_otel-configuration-of-instrumentation:
```
Instrumentation options are specified 
in the `OpenTelemetryCollector` custom resource.
```
Should this be instead like this?
```
Instrumentation options are specified 
in the `Instrumentation` custom resource.
```

Reply:
yes, it should be Instrumentation custom resource.